bigbluebutton-html5/imports/ui/components/video-preview/component.jsx

```diff
@@ -194,6 +199,20 @@ class VideoPreview extends Component {
     });
   }
 
+  componentDidUpdate() {
+    this.webcamListener();
+  }
+
+  async webcamListener() {
+    const { cameraAllowed, isInitialDeviceSet } = this.state;
+    const getDevices = await navigator.mediaDevices.enumerateDevices();
+    const hasVideoInput = getDevices.filter(device => device.kind === 'videoinput').length;
```

Comment:
The name of the variable is confusing, in my opinion. I would call it something like `videoInputLength`.

Reply:
Changed the value to boolean in last commit! Now the variable name makes more sense.